guides/common/modules/proc_synchronizing-a-custom-repository.adoc

```diff
@@ -0,0 +1,31 @@
+[id="Synchronizing_a_Custom_Repository_{context}"]
+= Synchronizing a Custom Repository
+
+Use this procedure to synchronize content from a custom repository on a connected {ProjectServer} and a disconnected {ProjectServer} via {ISS} (ISS) Network Sync.
```

Comment:
Would be good to add a note about the "why:" When using ISS Network Sync, Red Hat repositories are synced automatically, but custom repositories are not. They require performing the manual procedures outlined here.